Source/WebCore/platform/mediastream/RealtimeMediaSource.cpp

```diff
@@ -1137,7 +1137,7 @@ void RealtimeMediaSource::setLogger(const Logger& newLogger, const void* newLogI
 
 WTFLogChannel& RealtimeMediaSource::logChannel() const
 {
-    return LogWebRTC;
```

Comment:
Please revert this one, RealtimeMediaSource is about WebRTC. And for the others, MediaStream is more appropriate, IMHO.

Reply:
removed

Reply:
approved

Reply:
Actually, I think it would be more appropriate to get rid of the `MediaStream` log channel and change existing code to use `WebRTC` instead. Media stream is mostly used for WebRTC so I think it would be a benefit to not have to remember to enable two log channels to get all WebRTC-related logging.

Reply:
Ah OK, hm I don't have a strong opinion about this. So this PR would be invalid then.

Reply:
> Media stream is mostly used for WebRTC so I think it would be a benefit to not have to remember to enable two log channels to get all WebRTC-related logging.

It seems kinda confusing that one would need to enable WebRTC logging to get logging for codepaths that are not WebRTC specific.

If one makes a build without WebRTC support enabled at all then it's especially confusing as enabling logging at runtime for a compiletime disabled feature is very counterintuitive.

Maybe there's some way to have WebRTC logging automatically enable `LogMedia` logging as well?

Reply:
> It seems kinda confusing that one would need to enable WebRTC logging to get logging for codepaths that are not WebRTC specific.

I agree that it is somewhat confusing, but I think it is more confusing that enabling the `WebRTC` logging channel doesn't enable all of the logging in code that is essential to the WebRTC feature. 

There is relatively little logging to the `MediaStream` channel, and most "Media Stream" classes already use the `WebRTC` log channel (e.g. `MediaStream`, `MediaStreamTrack`, `UserMediaPermissionRequestManagerProxy`, etc), so I think it makes more sense to make it all use the it.

> If one makes a build without WebRTC support enabled at all then it's especially confusing as enabling logging at runtime for a compiletime disabled feature is very counterintuitive.

This is true, and unfortunate, but as there are more WebKit developers working on ports with WebRTC enabled I think it is acceptable for it to be slightly more confusing for those working on ports without it.